CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [0.22.4]
+
+### Fixed
+
+- Fixed the reported used bytes for any virtio-block request.
+- Fixed all virtio-block read/write operations to valid guest addresses
+  with buffer length of 0 to result in no-op.
```

Comment:
I think these would now result in error, initially they would be no-op. Am I missing something ?